### PR TITLE
Add missing hCaptcha attrs and general fixes

### PR DESCRIPTION
**Pull request by DSergiu**
Bumps version to: `0.3.0`

Closes https://github.com/hCaptcha/vue-hcaptcha/issues/29
Closes https://github.com/hCaptcha/vue-hcaptcha/issues/19
Closes https://github.com/hCaptcha/vue-hcaptcha/issues/21
Closes https://github.com/hCaptcha/vue-hcaptcha/issues/10
Closes https://github.com/hCaptcha/vue-hcaptcha/issues/5
Closes https://github.com/hCaptcha/vue-hcaptcha/issues/1

### CODE REVIEW

File: src/hcaptcha-script.js [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+/**
+ * Used to prevent loading api.js multiple times
+ * @type {string}
+ */
+const SCRIPT_ID = 'hcaptcha-api-script-id';
+
+const HCAPTCHA_LOAD_FN_NAME = '_hcaptchaOnLoad';
+
+class HcaptchaScript {
```
Comment (brdlyptrs) [inline src/hcaptcha-script.js:23]: Why make this a class, couldn't this be an async function?

Comment (DSergiu) [reply src/hcaptcha-script.js]: Sure thing. The point was to have a single instance of this so we can resolve promise once. Please check out the last commit.

File: src/__tests__/vue-hcaptcha.test.js [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+import {loadApiEndpointIfNotAlready} from '@/hcaptcha-script';
+import {nextTick} from 'vue';
+
+/*global describe, test, expect, beforeEach, jest*/
+
+jest.mock('@/hcaptcha-script', () => ({
+    loadApiEndpointIfNotAlready: jest.fn(() => Promise.resolve())
+}));
+
+describe('VueHCaptcha', () => {
```
Comment (brdlyptrs) [inline src/__tests__/vue-hcaptcha.test.js:13]: Any way to validate that the rendering (invisible, compact, dark mode) is correct?

File: examples/traditional-vue-app/README.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+Demo app to showcase usage of the vue web component.
+
+## Project setup
+```
+npm install
+```
+It uses latest version of `@hcaptcha/vue-hcaptcha`.
+
+When developing, make sure to link the local version of the component. Run followings:
+* on root: `npm link`
```
Comment (brdlyptrs) [inline examples/traditional-vue-app/README.md:12]: Are you installing the npm hosted version and showing that vs running the local repo one?

Comment (DSergiu) [reply examples/traditional-vue-app/README.md:12]: Indeed. The demo is about showing end usage, but can also be used when developing by doing a local npm link.

File: src/__tests__/vue-hcaptcha.test.js (hunk 2 above)
Comment (DSergiu) [reply src/__tests__/vue-hcaptcha.test.js:13]: There is a test `el renders after api loads and a widget id is set` which asserts that the passed properties are used in rendering. 

`src/__tests__/vue-hcaptcha.test.js` lines 113-118

File: examples/traditional-vue-app/README.md (hunk 3 above)
Comment (brdlyptrs) [reply examples/traditional-vue-app/README.md:12]: Isn't the point of pulling / forking this repository to work on it? The first thing I would do is run npm start thinking I can spin up code to work with.

Comment (DSergiu) [reply examples/traditional-vue-app/README.md]: Sure thing. Let's favor development over end usage example. 

Now it is possible to do `npm run serve` in one of the examples and a live-reload webpage is served which reflect changes of the `vue-hcaptcha` web component in real-time.

**Review by brdlyptrs — APPROVED** [review]

**Pull request merged**